### PR TITLE
Make the validator peerset more restrictive

### PR DESCRIPTION
**Pull request by timorl**
Also remove some ancient print debugging that somehow slipped by
reviewers. <_<"

### CODE REVIEW

File: finality-aleph/src/lib.rs [hunk 1]
```diff
@@ -54,10 +54,10 @@ pub fn peers_set_config(protocol: Protocol) -> sc_network::config::NonDefaultSet
 
     config.set_config = match protocol {
         Protocol::Validator => sc_network::config::SetConfig {
-            in_peers: 25,
+            in_peers: 0,
             out_peers: 0,
             reserved_nodes: Vec::new(),
-            non_reserved_mode: sc_network::config::NonReservedPeerMode::Accept,
+            non_reserved_mode: sc_network::config::NonReservedPeerMode::Deny,
         },
```
Comment (kostekIV) [inline finality-aleph/src/lib.rs:61]: would be cool to add a comment here on what it does. At first glance, it looks like we don't allow anyone to connect to us at all.

**Review by kostekIV — APPROVED** [review]

File: finality-aleph/src/lib.rs (hunk 1 above)
Comment (timorl) [reply finality-aleph/src/lib.rs:61]: And that is absolutely correct. We only connect based on authentications we get through the other protocol, which we use to make peers reserved.

**Comment by timorl** [comment]
Whoopsie, this might not actually work, I just noticed the way I tested this might have masked any issues.

**Comment by timorl** [comment]
Nvm this actually works as expected the python scripts/AWS were just acting up.

**Review by Yithis — APPROVED** [review]
Looks good to me

**Review by kostekIV — APPROVED** [review]

File: finality-aleph/src/lib.rs (hunk 1 above)
Comment (timorl) [reply finality-aleph/src/lib.rs:62]: Added comment.

**Pull request merged**